From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Tyler Veness <calcmogul@gmail.com>
Date: Sun, 30 Jul 2023 14:17:37 -0700
Subject: [PATCH 29/36] Remove DenseMap GTest printer test

LLVM modifies internal GTest headers to support it, which we can't do.
---
 llvm/unittests/ADT/DenseMapTest.cpp | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/llvm/unittests/ADT/DenseMapTest.cpp b/llvm/unittests/ADT/DenseMapTest.cpp
index 1f232d3046292c0da940ba4bef7d50604556e4c2..8d90d5afea79c619590cc32539e5124d02b1349c 100644
--- a/llvm/unittests/ADT/DenseMapTest.cpp
+++ b/llvm/unittests/ADT/DenseMapTest.cpp
@@ -737,11 +737,4 @@ TEST(DenseMapCustomTest, VariantSupport) {
   EXPECT_FALSE(DenseMapInfo<variant>::isEqual(Keys[2], Keys[2]));
 }
 
-// Test that gTest prints map entries as pairs instead of opaque objects.
-// See third-party/unittest/googletest/internal/custom/gtest-printers.h
-TEST(DenseMapCustomTest, PairPrinting) {
-  DenseMap<int, std::string_view> Map = {{1, "one"}, {2, "two"}};
-  EXPECT_EQ(R"({ (1, "one"), (2, "two") })", ::testing::PrintToString(Map));
-}
-
 } // namespace
